router.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const request = require('request');
+const router = express.Router();
+
```

Comment:
This doesn't affect this merge, but I think you're missing two routes. I have a PUT route on my buttons (you'll see it if you try clicking one of the cool, funny, or useful buttons) and Michael has a POST to edit the types of food associated with the restaurants (the edit button on his title).